packages/insight/src/components/search.tsx

```diff
@@ -153,8 +161,12 @@ const Search: FC<SearchProps> = ({borderBottom, id, setErrorMessage}) => {
   }
 
   const searchInputPlaceholder = useMemo(() => {
```

Comment:
Right now this `useMemo` only gets evaluated on change of `currency` and `network`. So if a user comes to the page on desktop screen and then they make it smaller, the placeholder would not change.

You should add `window.innerWidth` to the useMemo dependency array on Line 175.